packages/samples/headless-react/src/App.tsx

```diff
@@ -40,6 +43,24 @@ import {
 } from '@coveo/headless';
 import {bindSearchParametersToURI} from './components/search-parameter-manager/search-parameter-manager';
 
+const tabs = {
+  all: buildTab(engine, {
+    initialState: {isActive: true},
+    options: {expression: ''},
+  }),
+  messages: buildTab(engine, {
+    initialState: {isActive: false},
+    options: {expression: '@objecttype==Message'},
+  }),
+  confluence: buildTab(engine, {
+    initialState: {isActive: false},
```

Comment:
Let's remove initialState `false` to shorten the example. It's not necessary to specify since it is the default,